lib/linux_admin/common.rb

```diff
@@ -11,6 +17,7 @@ def run(cmd, options = {})
     end
 
     def run!(cmd, options = {})
+      AwesomeSpawn.logger = logger
```

Comment:
This is weird to do it on every call to run.  Not sure where you would do it otherwise though.